library/core/src/ptr/mod.rs

```diff
@@ -869,10 +871,12 @@ pub const unsafe fn read_unaligned<T>(src: *const T) -> T {
 /// ```
 #[inline]
 #[stable(feature = "rust1", since = "1.0.0")]
-pub unsafe fn write<T>(dst: *mut T, src: T) {
+#[rustc_const_unstable(feature = "const_ptr_write", issue = "none")]
```

Comment:
I know these did not have a tracking issue before, but that already lead to confusion -- would you mind creating one now?

Reply:
Done :) #86302